src/ServiceControl.Config/FodyWeavers.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Weavers GenerateXsd="false">
   <PropertyChanged />
-  <PropertyChanging />
+  <PropertyChanging EventInvokerNames="NotifyOfPropertyChange"/>
```

Comment:
https://github.com/Fody/PropertyChanging/wiki/EventInvokerSelectionInjection

Reply:
I tried to add `RaisePropertyChanged` but since it is an extension method it will not be picked up by fody